system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -555,7 +555,7 @@ pub mod bridging {
 				2,
 				[
 					GlobalConsensus(PolkadotNetwork::get()),
-					Parachain(kusama_runtime_constants::system_parachain::ASSET_HUB_ID),
+					Parachain(kusama_runtime_constants::system_parachain::AssetHubParaId::get().into()),
```

Comment:
If we bring back `ASSET_HUB_ID`, then we won't need to change everywhere `ASSET_HUB_ID` -> `AssetHubParaId::get().into()`

```suggestion
					Parachain(kusama_runtime_constants::system_parachain::ASSET_HUB_ID),
```